extensions/README.md

```diff
@@ -1,40 +1,37 @@
 # Jetpack Block Editor Extensions
 
-This directory lists extensions for the Block Editor, also known as Gutenberg, [that was introduced in WordPress 5.0](https://wordpress.org/news/2018/12/bebo/).
+This directory lists extensions for the Block Editor, also known as Gutenberg,
+[that was introduced in WordPress 5.0](https://wordpress.org/news/2018/12/bebo/).
 
 ## Extension Type
 
 We define different types of block editor extensions:
 
-- Blocks are available in the editor itself, and live in the `blocks` directory.
-- Plugins are available in the Jetpack sidebar that appears on the right side of the block editor. Those live in the `plugins` directory.
-
-When adding a new extension, add a new directory for your extension the matching directory.
+- Blocks are available in the editor itself.
+- Plugins are available in the Jetpack sidebar that appears on the right side of the block editor.
 
 ## Extension Structure
 
-Your extension should follow this structure:
+Extensions loosely follow this structure:
 
 ```
 .
-└── blockname/
-	└── blockname.php ← PHP file where the block and its assets are registered.
+└── block-or-plugin-name/
```

Comment:
In https://github.com/Automattic/jetpack/pull/11333 (unmerged) @ockham suggested separate folder for Plugins: `extensions/plugins/seo/seo.php`

Thoughts?

Reply:
Suppose extension could be both at the same time so it might not make sense to optimize this way?

Reply:
It may be easier for our build tools and for new contributors if everything was in one place?

Reply:
Does `/extensions/blocks` then make sense and should we just have `/extensions` ?

Reply:
I believe the things in `/blocks` are actually blocks.

#11640 includes additional extensions (the sidebar and category) that aren't blocks. They are not located under blocks.